### PR TITLE
updated ptp-slave role

### PR DESCRIPTION
**Pull request by mchwalisz**
It should now match the debian package.

### CODE REVIEW

File: deploy/roles/ptp-slave/tasks/main.yml [hunk 1]
```diff
@@ -14,15 +14,23 @@
...
-    dest: '/etc/systemd/system/'
+    src: '{{ item.src }}'
+    dest: '{{ item.dest }}'
     mode: 0644
   with_items:
-   - ptp4l.service
-   - ptp4l.path
-   - phc2sys.service
+   - { src: 'phc2sys_mands.conf', dest: '/etc/systemd/system/phc2sys.service.d/phc2sys_mands.conf' }
+   - { src: 'ptp_restart.conf', dest: '/etc/systemd/system/ptp4l.service.d/phc2sys.service' }
```
Comment (geissdoerfer) [inline deploy/roles/ptp-slave/tasks/main.yml]: It seems a bit odd to name an override file `.service`. Why not go with `.conf` as you do with `phc2sys`?

**Comment by geissdoerfer** [comment]
I was complaining about the duplicated code between the `shepherd-ptp` debian package and this ansible role offline. Wouldn't it make sense to at least have the systemd override files for ansible pulled in from the path of the debian package?

**Comment by mchwalisz** [comment]
I was thinking about the issue. We should, actually, replace most of the ansible role with tasks to install the debian package provided by us. This will unify the code and make it easier to keep track of both.

It of course depends on what we want to achieve with ansible playbooks.

File: deploy/roles/ptp-slave/tasks/main.yml (hunk 1 above)
Comment (mchwalisz) [reply deploy/roles/ptp-slave/tasks/main.yml]: fixed

**Comment by geissdoerfer** [comment]
Thanks. In fact, there is already an ansible playbook to install the debian packages, aimed at actual users. For developers it's inconvenient to build the debian packages first. I'd still like to offer a more direct way. On the other hand, once you actually develop, you'll probably anyway just use `make install` and so on to install just the components that you're currently working with. So maybe we just provide a playbook to setup all requirements like build tools and instructions how to setup a local development environment..

**Pull request merged**